.github/CODEOWNERS

```diff
@@ -1,8 +1,7 @@
 # Core maintainers:
```

Comment:
This section was meant as a "this person has administrative access to the `rust-windowing` repo", so maybe we should just rename it to that (the concept of "core maintainers" doesn't really make sense).

Reply:
Hm, that's fair, they have access to all the repos though...

Reply:
Maybe it's fine to do that that way, since core kind of makes sense, like owner access to all repos just happened to match with these people since they maintain more than project usually.

Reply:
Oh, didn't see that you've change the `rust-windowing` org structure to match this.

```suggestion
# Core maintainers (ping these for administrative tasks in `rust-windowing`):
```

Reply:
I'm not sure that we should have it here? It's not that discoverable. In general people should ping us on matrix. I've used core maintainers for the people who do public API now.

Reply:
Fair, perhaps we should just remove that section altogether.

Reply:
I think we should, and then when the backends will split we'll just add core stuff to get notified for winit-core and specific backends will be separate.